pyvo/registry/regtap.py

```diff
@@ -786,7 +799,7 @@ def get_interface(self, *,
                              "Otherwise, the search can be constrained by specifying a 'service_type' "
                              "or a 'keyword' to be found in the service description."
                              " You might also want to see all the available services"
-                             " with `pyvo.registry.regtap.RegistryResource.list_services()`.")
+                             " with `pyvo.registry.regtap.RegistryResource.list_interfaces()`.")
```

Comment:
For interactive use, I think I'd rather point people to .describe() -- but then space is precious here, so feel free to ignore this.